docs/ci-cd-environment/aws-support.md

```diff
@@ -62,7 +64,20 @@ aws ssm put-parameter \
   --key-id <your-kms-key-id>
 ```
 
-Note: if you want to use the default `aws/ssm` KMS, omit the `--key-id` argument.
+Note: you need `kms:Encrypt` permissions on the KMS key you use here. Alternatively, if you want to use the default `aws/ssm` KMS, omit the `--key-id` argument.
```

Comment:
I usually like to avoid the `Note:` at the bottom. Readers only read it once the above command fails.

I would rather suggest:

```
# Store the registration token

You can use the default SSM store to save the registration token, or you can
create a new SSM store for this agent type.

#### Use the default store

aws ssm put-parameter \
  --name <your-ssm-parameter-name> \
  --value "<your-agent-type-registration-token>" \
  --type SecureString


#### (Alternative) Create a new SSM store and store it there as a parameter

TODO: Document how to create one

and then:

aws ssm put-parameter \
  --name <your-ssm-parameter-name> \
  --value "<your-agent-type-registration-token>" \
  --type SecureString
  --key-id <ID>

```